proto/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -94,7 +94,7 @@ message GetLatestBlockRequest {}
 // GetLatestBlockResponse is the response type for the Query/GetLatestBlock RPC method.
 message GetLatestBlockResponse {
   .tendermint.types.BlockID block_id = 1;
-  .tendermint.types.Block   block    = 2;
+  Block                     block    = 2;
```

Comment:
So we can't change this, neither here nor on the TxService as marko suggested here https://github.com/cosmos/cosmos-sdk/pull/11390#issuecomment-1078978844, because both would be proto-breaking.

What we can do is simply add a new `proposer` string field to either or both of these 2 response types.

The alternative is to add a new field `sdk_block` on either or both of these 2 response types, which has the new Block type.